its/ruling/src/test/resources/expected/python-S1039.json

```diff
@@ -0,0 +1,17 @@
+{
+'project:django-2.2.3/django/core/handlers/base.py':[
+158,
+],
+'project:twisted-12.1.0/twisted/conch/scripts/cftp.py':[
+81,
+],
+'project:twisted-12.1.0/twisted/conch/scripts/ckeygen.py':[
+76,
+],
+'project:twisted-12.1.0/twisted/conch/scripts/conch.py':[
+153,
+],
+'project:twisted-12.1.0/twisted/conch/scripts/tkconch.py':[
+337,
+],
+}
```

Comment:
Given the names of the methods, I believe most (all?) of those issues are FPs ?
Those methods seem to be called in an `except` context for which we don't have the symbol usages (hence why we miss them?).
Do we plan to do something about those kind of FPs or just accept them and monitor the behavior on more projects?

Reply:
For the record, we finally decided to create a new dedicated _code smell_ rule.
Regarding symbol usages, we miss usages on `self` (e.g. `self.self.process_exception_by_middleware`). Ticket created for this: https://jira.sonarsource.com/browse/SONARPY-614